.github/workflows/tag-and-publish.yml

```diff
@@ -6,72 +6,38 @@ permissions:
 
 on:
   workflow_dispatch:
-    
-# This leads to too many releases :) 
-# on:
-#   push:
-#     branches: [ master ]
-#     paths:
-#       - "**.rs"
-#       - "**.go"
-#       - "**.toml"
-#       - "**.proto"
-
+    inputs:
+      sdk_version:
+        description: 'SDK version to tag and publish'
+        required: true
 env:
   PROTOC_VERSION: "26.1"
 
   
 jobs:
-  rust_build_and_test_linux:
-    name: Build and Test (Linux)
+  update_version:
+    name: "[Rust & Go SDK] Update SDK version"
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: rust
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: arduino/setup-protoc@v3
-        with:
-          version: ${{env.PROTOC_VERSION}}
-
-      - uses: actions-rs/toolchain@v1
-        with:
-            profile: minimal
-            toolchain: stable
-            override: true
-            components: rustfmt
-      - uses: taiki-e/install-action@nextest
-      - name: 'Build and test'
-        run: cargo nextest run --workspace --all-features
-
-  rust_build_and_test_windows:
-    name: "[Rust SDK] Unit Tests (Windows)"
-    runs-on: windows-latest
-    defaults:
-      run:
-        working-directory: rust
     steps:
       - uses: actions/checkout@v4
-      - uses: arduino/setup-protoc@v3
-        with:
-          version: ${{env.PROTOC_VERSION}}
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: de-vri-es/setup-git-credentials@v2
         with:
-            profile: minimal
-            toolchain: stable
-            override: true
-            components: rustfmt
-      - uses: taiki-e/install-action@nextest
-      - name: 'Build and test'
-        run: cargo nextest run --workspace --all-features
+          credentials: ${{secrets.GIT_CREDENTIALS}}
+      - name: Run protofetch tags updater
+        run: |
+          git config --global user.email "github@coralogix.com"
+          git config --global user.name "Github Actions"
+          cargo run --bin version_updater  -- -r ../.. -o ${{ github.repository }} -v ${{ github.event.inputs.sdk_version }}
+        working-directory: tools/release-management
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_READER_TOKEN: ${{ secrets.GH_READER_TOKEN }}
+          RUST_LOG: debug
 
   crates_io_publish:
     name: "[Rust & Go SDK] Tag and Publish to crates.io"
-    needs:
-      - rust_build_and_test_linux
-      - rust_build_and_test_windows
-
+    needs: update_version
```

Comment:
remove this since it will require to run the version updater RIGHT BEFORE the release. The Release (crates_io_publish) job has to be its own independent workflow (different file)

Reply:
Done